index.html

```diff
@@ -64,5 +64,214 @@ <h1 class="header__connect">
+        </p>
+        <ul class="tags">
+          <li class="html">
+            <a href="#" class="language">html</a>
+          </li>
+          <li class="css">
+            <a href="#" class="language">css</a>
+          </li>
+          <li class="javascript">
+            <a href="#" class="language">JavaScript</a>
+          </li>
+        </ul>
+        <button type="button" class="btn">
+          <a href="#" class="see__project">See Project</a>
+        </button>
+      </div>
+    </section>
+
+    <!-- /**
+        *! About Myself Section Here.
+        **/ -->
+
+    <section class="about__myself__container">
+      <div class="about__self">
+        <h1 class="about__myself__title">About<br> Myself</h1>
+        <p class="about__myself__description">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <h1 class="about__myself__connect">
+          <a href="#">LET'S CONNECT</a>
+        </h1>
+        <ul class="about__myself__socials">
+          <li class="github">
+            <a href="#">
+              <img src="./images/github.svg" alt="" />
+            </a>
+          </li>
+          <li class="linkedin">
+            <a href="#">
+              <img src="./images/Linkedin icon.svg" alt="" />
+            </a>
+          </li>
+          <li class="angellist">
+            <a href="#">
+              <img src="./images/angellist.svg" alt="" />
+            </a>
+          </li>
+          <li class="twitter">
+            <a href="#">
+              <img src="./images/twitter.svg" alt="" />
+            </a>
+          </li>
+          <li class="medium">
+            <a href="#">
+              <img src="./images/medium.svg" alt="" />
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="btn2">
+          <a href="#" class="resume">Get my resume</a>
+        </button>
+        <ul class="skills__list">
+          <li class="about__myself__language">
+            <h1 class="lang">Language</h1>
+          </li>
+          <li class="drop__down__icon">
+            <img src="./images/dropdown.svg" alt="" />
+          </li>
+        </ul>
+        <ul class="language-list">
+          <img src="./images/js.svg" alt="javascript-icon" />
+          <li class="code">JavaScript</li>
+        </ul>
+        <ul class="language-list">
+          <img src="./images/html.svg" alt="javascript-icon" />
+          <li class="code html2"  >HTML</li>
+        </ul>
+        <ul class="language-list">
+          <img src="./images/css.svg" alt="javascript-icon" />
+          <li class="code css2">CSS</li>
+        </ul>
+        <ul class="first-frame">
+          <li class="framework">
+            <h1>Framework</h1>
+          </li>
+          <li class="right__icon">
```

Comment:
- [ ] Good job here but I think the technology categories ("Languages", "Frameworks" and "Skills") should be in one `ul`, and then each of these three should be in each `li`, taking multiple `ul`s where we do not need them is not good practice. 
- [ ] Then, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be also a list `ul` and each of them a `li`. 
so in total you only need two `ul` in this section. kindly consider reorganizing your `ul`s and `li`s in this section following best practices and best semantics.